### ISSUE TITLE
Feature Request: Add privacy policy

### ISSUE THREAD

**Issue by ngdenco** [deanoemcke/thegreatsuspender#786]
In the reviews for the extension itself, there is a comment stating this:

> WARNING: THIS EXTENSION GATHERS YOUR BROWSING HISTORY AND SElLS IT TO 3RD PARTY ENTITIES

However I cannot find any evidence of this claim, whether to back it or disprove it.

In the 'about' page, could there be information added regarding your privacy policy and whether or not information is gathered for third parties?

**Comment by deanoemcke** [comment]
That comment is erroneous. This extension does not collect or sell browsing history. I have commented to the person on the chrome webstore.

The extension DOES require permission to your chrome history, as this is required for some of the functionality of the app. However, it does not store this history, and does not profit from it in any way.

For more information on why the extension needs access to your history please see this gitHub thread: https://github.com/deanoemcke/thegreatsuspender/issues/213

Also, here is a link to the privacy policy for the extension: https://greatsuspender.github.io/privacy

**Comment by ngdenco** [comment]
Perfect, thanks. I would recommend a link to the policy in your "about" page in the app, perhaps as well as on the actual store page. 
And probably as a reply to that comment :)

**Comment by deanoemcke** [comment]
Yes, I agree the 'about' page would be a good place for it.
The privacy policy is already linked in the chrome webstore page.

**Comment by ngdenco** [comment]
I agree with the overall sentiment. I have a few "counterpoints" to the situation, however:

- The standard user will not look through source code, nor know what to look for
- I am a Java developer and looked through a few parts of the code, and admittedly was lost in finding anything. Afterall, it's not like the code doing this is called "third-party-transfer". Sifting through open source isn't super friendly to even developers, but of course it's eons better than closed.
- You are attributing selling data to third parties to maliciousness. There are tools who actively do sell information to third parties, and let you know they may do that in their clauses. Even in the case of open source, they could still be doing this. It could easily be the business model of the program, and not hidden from users. With that being said I wanted to know a yes or no answer to the original question, regardless of morality of the action.

**Comment by CollinChaffin** [comment]
Certainly agree with your points in fact it is funny you say that as I was posting over a EIGHTEEN MONTHS ago in many places including different repos here on Github (https://github.com/StylishThemes/StackOverflow-Dark/issues/44) because a legitimate marketing company is who quietly purchased Stylish, and their privacy policy was immediately altered to flat out state they could(would) do as much, so I was surprised when folks were so surprised to find out they then took advantage of it.  And, in agreement that is why the policies are important at least those that are honest and legit enough to post them as they do demonstrate intent.  IMO the TGS policy is out there and pretty straight forward and with their totally un-obfuscated source here I personally am not at all worried.  :wink:
